bot.py

```diff
@@ -0,0 +1,36 @@
+import discord
+import json
+import asyncio
+import datetime
+import time
+from discord import app_commands
+from discord.ext import commands
+from game_manager import refresh_game_list
+from commands import setup_commands
+from utils import load_config
+
+# Load Config
+config = load_config()
+
+# Setup Discord Bot
+intents = discord.Intents.default()
+client = commands.Bot(command_prefix="!", intents=intents)
+
+@client.event
+async def on_ready():
+    print(f'✅ Logged in as {client.user}')
+    await setup_commands(client)
+    client.loop.create_task(background_task())
+
+async def background_task():
+    """Periodically refreshes the game list with dynamically loaded config."""
+    while True:
+        config = load_config()  # Reload config before every cycle
+        await asyncio.sleep(config["refresh_time"])
+        await refresh_game_list(client)
+
+# Run Bot
+with open('./discord_bot_token', 'r') as file:
```

Comment:
Double quotes here too.
```suggestion
with open("./discord_bot_token", "r") as file:
```